config/config-network.yaml

```diff
@@ -182,3 +183,14 @@ data:
     # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
     #       for now. Use with caution.
     internal-encryption: "false"
+
+    # internal-trust indicates whether internal traffic is encrypted and mutual trust is achieved.
+    # Using internal-trust protect against MITM attacks
```

Comment:
I think getting SANs fixed to be distinct per namespace also prevents MITM attacks, no?  (Now, it doesn't authenticate _clients_, so you can potentially bypass the KIngress implementation, but that's not a MITM attack...)

Reply:
I agree

Reply:
For this SANs need to be meaningful. I.e. as a minimum a different SAN per namespace